Scuti/Scripts/UI/Implementations/Modals/LogoutView.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Scuti
+{
+
+    public class LogoutView : View
+    {
+        [Header("Interaction")]
+        [SerializeField] Button acceptButton;
+        [SerializeField] Button closeButton;
+
+        System.Action<bool> m_Callback;
+
+        protected override void Awake()
+        {
+            base.Awake();
+            Init();
+        }
+
+        void Init()
+        {
+            onClosed.AddListener(() => m_Callback = null);
+            if (acceptButton) acceptButton.onClick.AddListener(OnClickAccept);
+            if (closeButton) closeButton.onClick.AddListener(OnClickClose);
+        }
+
+        public void Show(System.Action<bool> callback)
+        {
+            m_Callback = callback;
+            Open();
+        }
+
+        void OnClickAccept()
+        {
+            Close();
+            m_Callback?.Invoke(true);
+        }
+
+        void OnClickClose()
+        {
+            Close();
+            m_Callback?.Invoke(false);
+        }
```

Comment:
@Jpablogs8 doesn't this mean the callback will get called whether it is accepted or not? Should callback only be onClickClose?

Reply:
@markgrossnickle According to how I saw it working from Navigation you needed to know both cases to show the offers again. That's why the true or false